tests/test.nim

```diff
@@ -48,8 +48,8 @@ var counter {.global.}: Atomic[int]
 when defined(windows):
   proc noop(c: C): C {.cpsMagic.} =
     sleep:
-      when defined(danger):
-        1
+      when defined(danger) and false: # I'm the only person using windows
+        1                             # and this is annoying
```

Comment:
If we don't need to sleep, feel free to take this stuff out...  I did it this way because we cannot sleep for less than 1ms on Windows. :shrug: